sharepoint/sharepoint-ps/sharepoint-online/Get-SPOMalwareFile.md

```diff
@@ -0,0 +1,65 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://docs.microsoft.com/powershell/module/sharepoint-online/Get-SPOMalwareFile
+applicable: SharePoint Online
+title: Get-SPOMalwareFile
+schema: 2.0.0
+author: vinandi
+ms.author: vinandi
+ms.reviewer:
+---
+
+# Get-SPOMalwareFile
+
+## SYNOPSIS
+
+Extracts and displays the malware related information of an infected file stored in SharePoint.
+
+## SYNTAX
+
+```powershell
+Get-SPOMalwareFile -FileUri <String> [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+The `Get-SPOMalwareFile` cmdlet runs on a single file. If the file is malware-infected then it returns the siteURL, filePath, malwareStatus flag and malwareInfo (if applicable). You must be a SharePoint Online administrator or Global Administrator to run the `Get-SPOMalwareFile` cmdlet. Note that this cmdlet does not work on files that are not malware-infected.
+For permissions and the most current information about Windows PowerShell for SharePoint Online, see the online documentation at [Intro to SharePoint Online Management Shell](/powershell/sharepoint/sharepoint-online/introduction-sharepoint-online-management-shell).
+
+## EXAMPLES
+
+### EXAMPLE 1
+
+```powershell
+ Get-SPOMalwareFile -FileUri "https://contoso.com/sites/Marketing/Shared Documents/Doc1.docx"
+```
+
+## PARAMETERS
+
+### -FileUri
+
+Full URL for the file.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+Applicable: SharePoint Online
+
+Required: True
+Position: Named
+Default value: None
+```
+
+## NOTES
+
+The `Get-SPOMalwareFile` cmdlet returns an object of the type `SPOMalwareFile`. To get the file stream of the infected file, pass this object as a parameter to the `Get-SPOMalwareFileContent` cmdlet.
+
+## RELATED LINKS
+
+[Get-SPOMalwareFileContent](Get-SPOMalwareFileContent.md)

```

Comment:
I think you need to have this checked in first, or include it in this PR

Reply:
included in this pr